dcos-metrics-collector.go

```diff
@@ -39,7 +44,7 @@ type AgentConfig struct {
 // Config ...
 type Config struct {
 	HTTPProfiler  bool   `yaml:"http_profiler"`
-	KafkaProducer bool   `yaml:"kafka_producer"`
+	KafkaProducer bool   `yaml:"kafka_producer_enabled"`
```

Comment:
Would it be worthwhile to change this to a more generic Producer, and have the user specify one? As opposed to enabling / disabling the Kafka producer itself.